torch/csrc/jit/passes/quantization/helper.h

```diff
@@ -44,6 +44,9 @@ TORCH_API bool isScalar(Value* v);
 // Check if value is the input of the graph
 TORCH_API bool hitGraphInput(Value* value);
 
+// Return the module name that corresponds to the value.
+TORCH_API c10::optional<std::string> get_module_name(Value* value);
```

Comment:
should we use the same naming convention like other functions? I just noticed this problem, probably the filter functions should follow the name convention as well. but we can do that later.